src/services/fileServices/MdPageServices/PageService.ts

```diff
@@ -142,19 +142,22 @@ export class PageService {
           if (rawPath.length > 2) {
             return errAsync(new NotFoundError())
           }
```

Comment:
Should we remove this part since we still return a NotFoundError later in line 160 if the two if-branches are not satisfied?

Reply:
actually i left it in just in case we wanted to return a more specific error in the future but yea, i htink this is more of code bloat for now so i'll just remove it